packages/platform/atoms/booker-embed/BookerEmbedPlatformWrapper.tsx

```diff
@@ -0,0 +1,15 @@
+import { CalProvider } from "../../atoms/cal-provider/CalProvider";
+import type { BookerPlatformWrapperAtomProps } from "../booker/BookerPlatformWrapper";
+import { BookerEmbed } from "./BookerEmbed";
+
+export const BookerEmbedPlatformWrapper = (props: BookerPlatformWrapperAtomProps) => {
```

Comment:
Would it be possible as a part of this PR add the booker embed in the examples base app, so that it is there and we can continiously test it?

Reply:
Trying to think if I should add this in a separate page or just replace the existing booker with the new embed atom?